components/omega/doc/design/SubmesoscaleEddies.md

```diff
@@ -0,0 +1,345 @@
+
+1.  **Reduced Spurious Restratification:** The modified scheme substantially reduces the excessive restratification seen with the original FK08 scheme, particularly in the high latitudes of the Southern Hemisphere during winter.
+2.  **Improved Mixed-Layer Depths:** As a result, the model simulates deeper and more realistic winter mixed-layer depths in the Southern Ocean, which is a long-standing bias in many climate models.
+3.  **Impacts on Water Mass Formation:** The deeper mixed layers enhance the formation of Subantarctic Mode Water (SAMW), improving the representation of water mass properties and the transport of heat and carbon into the ocean interior.
+4.  **Teleconnections:** The changes in the Southern Ocean have far-reaching impacts, including a strengthening of the Atlantic Meridional Overturning Circulation (AMOC) due to modified water mass pathways.
+
+Overall, the Uchida et al. study demonstrates that the physically-based constraints introduced by Bodner et al. lead to a more realistic and stable climate simulation.
+
+## 2. Requirements
+
+### 2.1 Requirement: Buoyancy calculation must be modular
+
+To allow for easier addition of a future mesoscale eddy closure ([e.g. Mak et al 2018](https://journals.ametsoc.org/view/journals/phoc/48/10/jpo-d-18-0017.1.xml) and/or [Korn et al 2018](https://www.sciencedirect.com/science/article/pii/S1463500318301859)) the buoyancy calculation and horizontal gradient that is required for the submesoscale eddy parameterization ([](#submeso-definition)) should be usable for mesoscale eddy closures.  For a non-boussinesq model, the buoyancy is defined as
+
+$$
+b \equiv g \frac{\rho - \rho_0}{\rho}
+$$
+
+where $\rho_0$ is a constant reference density.  The vertical mixed layer average, where $H$ is the mixed layer depth, in pseudo-height coordinates is given by
+
+$$
+b_{ml} = \frac{1}{H} \int_{-H}^0 g\frac{\rho - \rho_0}{\rho} d \tilde{z}
+$$
+
+This can be rewritten as
+
+$$
+b_{ml} = \frac{1}{H}\int_{-H}^0 g d\tilde{z} - \frac{1}{H}\int_{-H}^0 g \frac{\rho_0}{\rho} d \tilde{z}
+$$
+
+since $g$ is constant, we can write
+
+$$
+b_{ml} = g  - \frac{1}{H}\int_{-H}^0 g \frac{\rho_0}{\rho} d \tilde{z} = g - \frac{1}{H}\int_{-H}^0 g \rho_0 \alpha
+$$
+
+where $\alpha$ is the specific volume.
+
+### 2.2 Requirement: Buoyancy calculation must account for tilted pressure coordinates
+
+As in MPAS-Ocean, the horizontal buoyancy gradient calculation must supported tilted coordinates.  In the psuedo height coordinate for Omega, the horizontal gradient of buoyancy for a tilting surface ($\tilde{z}$) is
+
+$$
+\nabla b = \nabla_h b + \frac{\partial b}{\partial \tilde{z}} \nabla_h \tilde{z}
```

Comment:
Can we say more about how this will be used in the Algorithm section? The previous section makes it seem (to me) like we will be averaging the buoyancy over the mixed layer and then taking the horizontal gradient